tests/testsuites/imfile-wildcards-dirs-multi4.conf

```diff
@@ -0,0 +1,35 @@
+$IncludeConfig diag-common.conf
+$WorkDirectory test-spool
+
+/* Filter out busy debug output, comment out if needed */
+global(
```

Comment:
Is it really a good idea to disable debug log parts by default?

Reply:
Yes for these imfile tests. Debug log gets very huge with 99% unuseful informationen.